key-attestation/java-example/src/test/java/com/fortanix/keyattestationstatementverifier/VerifyTest.java

```diff
@@ -101,4 +110,88 @@ public void verifyStatementFullCheck() throws Exception {
 
         Verify.verify(decodedResponse, trustedRootCert, true);
     }
+
+    /**
+     * This test tests the full process of creating a RSA key, get RSA key's key
+     * attestation statement and finally verify it.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void verifyStatementFullCheckOnlineAMER() throws Exception {
+        // Because here we use an APP API key so we could skip many steps: select an account / create a group
+        String appApiKeyString = System.getenv(JAVA_CI_AMER_APP_API_KEY);
+        String authString = "Basic " + appApiKeyString;
+        // Create a RSA key
+        String generateKeyUrl = FORTANIX_AMER_SAAS_SERVER_URL + "/crypto/v1/keys";
+        String newRsaKeyName = UUID.randomUUID().toString();
+        String generateKeyRequest = String.format(
+                "{\"name\":\"%s\",\"description\":\"\",\"obj_type\":\"RSA\",\"key_ops\":[\"APPMANAGEABLE\",\"SIGN\",\"VERIFY\"],\"key_size\":2048,\"pub_exponent\":65537,\"expirationDate\":null,\"enabled\":true,\"rsa\":{\"encryption_policy\":[{\"padding\":{\"OAEP\":{\"mgf\":{\"mgf1\":{}}}}}],\"signature_policy\":[{\"padding\":{\"PKCS1_V15\":{}}},{\"padding\":{\"PSS\":{\"mgf\":{\"mgf1\":{}}}}}]}}",
+                newRsaKeyName);
+        System.out.println(
+                String.format("Creating a new RSA key named '%s' through %s ...", newRsaKeyName, generateKeyUrl));
+        String generateKeyResponseString = sendHttpRequest(generateKeyUrl, "POST", generateKeyRequest, authString);
```

Comment:
Why not use a proper DSM client for this?

Reply:
Fixed